client/data/promote-post/use-promote-post-campaigns-query-paged.ts

```diff
@@ -0,0 +1,140 @@
+import { useInfiniteQuery } from '@tanstack/react-query';
+import { requestDSP } from 'calypso/lib/promote-post';
+import { SearchOptions } from 'calypso/my-sites/promote-post-i2/components/search-bar';
+
+export enum CampaignStatus {
+	ALL = -1,
+	TODO0 = 0,
+	TODO1 = 1,
+	TODO2 = 2,
+}
+
+export const AudienceListKeys = {
+	topics: 'topics',
+	countries: 'countries',
+	devices: 'devices',
+	OSs: 'OSs',
+};
+
+export type AudienceList = {
+	[ key in keyof typeof AudienceListKeys ]: string;
+};
+
+export type Campaign = {
+	content_config: {
+		clickUrl: string;
```

Comment:
It would be nice if we could unify the format for these fields. The rest of the fields use the snake case format, expect for the two that use camel case.

Reply:
Count with it! This information will be used in the new campaign individual page. I'll normalize the fields there!